testsuite/forge-cli/src/main.rs

```diff
@@ -452,9 +452,13 @@ fn single_test_suite(test_name: &str) -> Result<ForgeConfig<'static>> {
             })),
         "config" => config.with_network_tests(vec![&ReconfigurationTest]),
         "network_partition" => config.with_network_tests(vec![&NetworkPartitionTest]),
-        "network_latency" => config
-            .with_network_tests(vec![&NetworkLatencyTest])
-            .with_success_criteria(SuccessCriteria::new(4000, 10000, true, None, None)),
+        "three_region_simulation" => config
+            .with_initial_validator_count(NonZeroUsize::new(20).unwrap())
+            .with_initial_fullnode_count(10)
+            .with_emit_job(EmitJobRequest::default().mode(EmitJobMode::ConstTps { tps: 5000 }))
+            .with_network_tests(vec![&ThreeRegionSimulationTest])
+            // TODO(rustielin): tune these success critiera after we have a better idea of the test behavior
+            .with_success_criteria(SuccessCriteria::new(1000, 100000, true, None, None)),
```

Comment:
Why so low TPS criteria? We can do more than 3k right?

Reply:
bumped to 3k, but we can settle on a more reasonable number after we run it continuously and get a feel of the test result variance